app/models/incremental_refresh_upload_task.rb

```diff
@@ -8,8 +8,8 @@ def post_upload_task
   def dispatch
     super
 
+    reload
```

Comment:
@hsong-rh Do you need one for check availability too?

Reply:
@mkanoor it's there already: https://github.com/RedHatInsights/catalog_inventory-api/blob/master/app/models/check_availability_task.rb#L18